.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "java.compile.nullAnalysis.mode": "automatic",
+    "java.configuration.updateBuildConfiguration": "interactive"
```

Comment:
Please remove this file

Reply:
done

Reply:
i run --> ```git rm --cached ~/Desktop/workStation/cryostat/.vscode/settings.json ``` the way you showed me the other day and then added this on my .gitconfig
``` [user]
        email = atali@redhat.com
        name = Atif Ali
        signingkey = 44ECBBDDF9A888E0
[commit]
        gpgsign = true
[core]
        excludesFile = ~/.gitignore
```
and i have that on my .gitignore 
```
 .vscode
*.swp
``` 
but still it added the file.. did i do any thing wrong on this set up?

Reply:
It's probably case sensitive. I have `~/.gitconfig` contents:

```
...
[core]
    autocrlf = input
    excludesfile = ~/.gitignore
...
```